### PR TITLE
Feature structure-2

### PR DESCRIPTION
**Pull request by kimothokamau**
This pull request contains: 
- Project setup using webpack
- Todo list structure

### CODE REVIEW

**Comment by kimothokamau** [comment]
This pull request contains: 
- Project setup using webpack
- Todo list structure

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] As seen in the original design kindly add the three-dots-icon and the enter icon.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import displayTodos from './frontend.js';
+import { todos } from './data.js';
+
+const todoapp = () => {
+  let todoLs = JSON.parse(localStorage.getItem('todos'));
+
+  if (todoLs === null) {
+    todoLs = todos;
+  }
+  displayTodos(todos);
```
Comment (mikenath223) [inline src/index.js:11]: - [x] As seen in the original design please add the `three-dots` icon and the `enter` icon.

| Original | Yours | 
| ----- | ----- |
|  |  |

**Comment by kimothokamau** [comment]
- Add the sync, add and delete icons

**Review by ZawadiSumba66 — APPROVED** [review]
#### Hi @kimothokamau 👋 👋 , 
## STATUS: APPROVED 👍 
All the changes requested by the previous reviewer have been implemented 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**